department-of-veterans-affairs/caseflow
# Veterans Incorrectly Mapping to Yakima Location

Resolves #9721 

### Description
After reviewing the available hearing locations calculated for the examples in the ticket I confirmed the distances were properly calculated. The issue was in the UI where the code picks the first entry in the array of available hearing locations that arrive to the UI. Unfortunately the array is not always sorted in proper ascending order by distance.

Fix was to resort in the UI prior to selecting the first entry to display in the Assign Hearings tabs.

### Acceptance Criteria

### Testing Plan
1. Go to Schedule Veterans
2. Select the Seattle Regional Office
3. Select the Legacy Veterans Waiting tab
4. Confirm the veterans in the ticket shown as example now show the proper alternate hearing location.
